ECMA2Yaml/ECMAHelper/YamlGenerators/SDPTOCGenerator.cs

```diff
@@ -91,9 +95,45 @@ private static TocViewModel GenerateTocItemsForMembers(Models.Type t)
                     Name = m.DisplayName
                 };
                 tocEntry.Metadata["type"] = m.ItemType.ToString();
+                if (!IsMonikersEqual(t.Monikers, m.Monikers))
+                {
+                    tocEntry.Metadata[OPSMetadata.Monikers] = m.Monikers.ToArray();
+                }
                 items.Add(tocEntry);
             }
             return items;
         }
+
+        private static bool IsMonikersEqual(HashSet<string> tMonikers, HashSet<string> mMonikers)
```

Comment:
Since here are two HashSets, there's a [SetEquals](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1.setequals?view=netframework-4.8) method that you can directly use to compare them